Fix npm install test failure

```
npm ERR! path /home/circleci/tasit/decentraland/node_modules/react-native-safe-area-view
npm ERR! code EISGIT
npm ERR! git /home/circleci/tasit/decentraland/node_modules/react-native-safe-area-view: Appears to be a git repo or submodule.
npm ERR! git     /home/circleci/tasit/decentraland/node_modules/react-native-safe-area-view
npm ERR! git Refusing to remove it. Update manually,
npm ERR! git or move it out of the way first.
```

--- a/.circleci/config.yml
+++ b/.circleci/config.yml
@@ -20,20 +20,20 @@ jobs:
           command: "sudo npm install -g npm@latest"
 
       # Download and cache dependencies
-      - restore_cache:
-          keys:
-            - dependency-cache-{{ checksum "package.json" }}
-            # fallback to using the latest cache if no exact match is found
-            - dependency-cache-
+      #- restore_cache:
+      #    keys:
+      #      - dependency-cache-{{ checksum "package.json" }}
+      #      # fallback to using the latest cache if no exact match is found
+      #      - dependency-cache-
 
       - run:
           name: npm install
           command: npm install
 
-      - save_cache:
-          paths:
-            - ./node_modules
-          key: dependency-cache-{{ checksum "package.json" }}
+      #- save_cache:
+      #    paths:
+      #      - ./node_modules
+      #    key: dependency-cache-{{ checksum "package.json" }}
 
       # run tests!
       - run:

--- a/.gitignore
+++ b/.gitignore
@@ -1,2 +1,1 @@
 .DS_Store
-package-lock.json

--- a/decentraland/helpers/index.js
+++ b/decentraland/helpers/index.js
@@ -100,6 +100,7 @@ export const fundAccountWithEthers = accountAddress => {
   gnosisSafeContract.setSigners([gnosisSafeOwner]);
 
   const action = gnosisSafeContract.transferEther(accountAddress, SMALL_AMOUNT);
+  console.info("Built the action to transfer ETH to ephemeral account");
   return action;
 };
 

--- a/decentraland/helpers/index.js
+++ b/decentraland/helpers/index.js
@@ -87,7 +87,7 @@ export const approveManaSpending = fromAccount => {
   const contracts = getContracts();
   const { manaContract, marketplaceContract } = contracts;
   const toAddress = marketplaceContract.getAddress();
-  manaContract.setWallet(fromAccount);
+  manaContract.setAccount(fromAccount);
 
   const action = manaContract.approve(toAddress, HALF_MILLION);
   return action;
@@ -96,7 +96,7 @@ export const approveManaSpending = fromAccount => {
 export const fundAccountWithEthers = accountAddress => {
   const contracts = getContracts();
   const { gnosisSafeContract } = contracts;
-  gnosisSafeContract.setWallet(gnosisSafeOwner);
+  gnosisSafeContract.setAccount(gnosisSafeOwner);
   gnosisSafeContract.setSigners([gnosisSafeOwner]);
 
   const action = gnosisSafeContract.transferEther(accountAddress, SMALL_AMOUNT);
@@ -106,7 +106,7 @@ export const fundAccountWithEthers = accountAddress => {
 export const fundAccountWithMana = accountAddress => {
   const contracts = getContracts();
   const { manaContract, gnosisSafeContract } = contracts;
-  gnosisSafeContract.setWallet(gnosisSafeOwner);
+  gnosisSafeContract.setAccount(gnosisSafeOwner);
   gnosisSafeContract.setSigners([gnosisSafeOwner]);
 
   const action = gnosisSafeContract.transferERC20(

--- a/decentraland/package.json
+++ b/decentraland/package.json
@@ -37,7 +37,7 @@
     "react-navigation": "^3.0.9",
     "react-redux": "^6.0.0",
     "redux": "^4.0.1",
-    "tasit-sdk": "0.0.17"
+    "tasit-sdk": "0.0.18"
   },
   "devDependencies": {
     "babel-eslint": "^10.0.1",

--- a/decentraland/package.json
+++ b/decentraland/package.json
@@ -52,7 +52,7 @@
     "eslint-plugin-prettier": "^3.0.0",
     "eslint-plugin-react": "^7.11.1",
     "eslint-plugin-react-native": "^3.5.0",
-    "expo-cli": "^2.15.4",
+    "expo-cli": "^2.17.1",
     "jest": "^24.1.0",
     "jest-expo": "^32.0.0",
     "jest-prop-type-error": "^1.1.0",

--- a/decentraland/screens/BuyLandScreen/BuyLandScreen.js
+++ b/decentraland/screens/BuyLandScreen/BuyLandScreen.js
@@ -65,7 +65,8 @@ export class BuyLandScreen extends React.Component {
     const onSuccess = async () => {
       // TODO: This function should be called inside of the eventListener
       // that catches the safeExecuteOrder successful event.
-      await action.waitForNonceToUpdate();
+      await action.waitForOneConfirmation();
+      // TODO: Change me to pub/sub style
       const actionId = await action.getId();
 
       updateUserActionStatus({ actionId, status: SUCCESSFUL });
@@ -116,7 +117,7 @@ export class BuyLandScreen extends React.Component {
       const fingerprint =
         type === ESTATE ? await estateContract.getFingerprint(assetId) : "0x";
 
-      marketplaceContract.setWallet(account);
+      marketplaceContract.setAccount(account);
       const action = marketplaceContract.safeExecuteOrder(
         nftAddress,
         `${assetId}`,

--- a/decentraland/screens/BuyLandScreen/BuyLandScreen.js
+++ b/decentraland/screens/BuyLandScreen/BuyLandScreen.js
@@ -84,7 +84,7 @@ export class BuyLandScreen extends React.Component {
     showInfo(`Buying the ${typeDescription.toLowerCase()}...`);
 
     const action = await _executeOrder(landForSale, account, onError);
-    action.send();
+    await action.send();
 
     // Optimistic UI update
     removeLandForSale(landForSale);

--- a/decentraland/screens/BuyLandScreen/BuyLandScreen.js
+++ b/decentraland/screens/BuyLandScreen/BuyLandScreen.js
@@ -84,6 +84,7 @@ export class BuyLandScreen extends React.Component {
     showInfo(`Buying the ${typeDescription.toLowerCase()}...`);
 
     const action = await _executeOrder(landForSale, account, onError);
+    action.send();
 
     // Optimistic UI update
     removeLandForSale(landForSale);

--- a/decentraland/screens/EthereumSignUpScreen/EthereumSignUpScreen.js
+++ b/decentraland/screens/EthereumSignUpScreen/EthereumSignUpScreen.js
@@ -54,7 +54,9 @@ export class EthereumSignUpScreen extends React.Component {
       const fundWithEthers = async accountAddress => {
         const action = fundAccountWithEthers(accountAddress);
         action.send();
+        console.info("Sent the ETH transfer action");
         const actionId = await action.getId();
+        console.info({ actionID });
         updateActionIdForAccountCreationStatus({
           status: FUNDING_WITH_ETH,
           actionId,

--- a/decentraland/screens/EthereumSignUpScreen/EthereumSignUpScreen.js
+++ b/decentraland/screens/EthereumSignUpScreen/EthereumSignUpScreen.js
@@ -53,6 +53,7 @@ export class EthereumSignUpScreen extends React.Component {
 
       const fundWithEthers = async accountAddress => {
         const action = fundAccountWithEthers(accountAddress);
+        action.send();
         const actionId = await action.getId();
         updateActionIdForAccountCreationStatus({
           status: FUNDING_WITH_ETH,
@@ -66,6 +67,7 @@ export class EthereumSignUpScreen extends React.Component {
 
       const fundWithMana = async accountAddress => {
         const action = fundAccountWithMana(accountAddress);
+        action.send();
         const actionId = await action.getId();
         updateActionIdForAccountCreationStatus({
           status: FUNDING_WITH_MANA,
@@ -79,6 +81,7 @@ export class EthereumSignUpScreen extends React.Component {
 
       const approveMarketplace = async account => {
         const action = approveManaSpending(account);
+        action.send();
         const actionId = await action.getId();
         updateActionIdForAccountCreationStatus({
           status: APPROVING_MARKETPLACE,

--- a/decentraland/screens/EthereumSignUpScreen/EthereumSignUpScreen.js
+++ b/decentraland/screens/EthereumSignUpScreen/EthereumSignUpScreen.js
@@ -56,7 +56,7 @@ export class EthereumSignUpScreen extends React.Component {
         action.send();
         console.info("Sent the ETH transfer action");
         const actionId = await action.getId();
-        console.info({ actionID });
+        console.info({ actionId });
         updateActionIdForAccountCreationStatus({
           status: FUNDING_WITH_ETH,
           actionId,

--- a/decentraland/screens/EthereumSignUpScreen/EthereumSignUpScreen.js
+++ b/decentraland/screens/EthereumSignUpScreen/EthereumSignUpScreen.js
@@ -53,7 +53,7 @@ export class EthereumSignUpScreen extends React.Component {
 
       const fundWithEthers = async accountAddress => {
         const action = fundAccountWithEthers(accountAddress);
-        action.send();
+        await action.send();
         console.info("Sent the ETH transfer action");
         const actionId = await action.getId();
         console.info({ actionId });
@@ -69,7 +69,7 @@ export class EthereumSignUpScreen extends React.Component {
 
       const fundWithMana = async accountAddress => {
         const action = fundAccountWithMana(accountAddress);
-        action.send();
+        await action.send();
         const actionId = await action.getId();
         updateActionIdForAccountCreationStatus({
           status: FUNDING_WITH_MANA,
@@ -83,7 +83,7 @@ export class EthereumSignUpScreen extends React.Component {
 
       const approveMarketplace = async account => {
         const action = approveManaSpending(account);
-        action.send();
+        await action.send();
         const actionId = await action.getId();
         updateActionIdForAccountCreationStatus({
           status: APPROVING_MARKETPLACE,

--- a/decentraland/screens/EthereumSignUpScreen/EthereumSignUpScreen.js
+++ b/decentraland/screens/EthereumSignUpScreen/EthereumSignUpScreen.js
@@ -58,7 +58,8 @@ export class EthereumSignUpScreen extends React.Component {
           status: FUNDING_WITH_ETH,
           actionId,
         });
-        await action.waitForNonceToUpdate();
+        await action.waitForOneConfirmation();
+        // TODO: Change me to pub/sub style
         showInfo(`Account funded with ETH`);
         setAccountCreationStatus(FUNDING_WITH_MANA_AND_APPROVING_MARKETPLACE);
       };
@@ -70,7 +71,8 @@ export class EthereumSignUpScreen extends React.Component {
           status: FUNDING_WITH_MANA,
           actionId,
         });
-        await action.waitForNonceToUpdate();
+        await action.waitForOneConfirmation();
+        // TODO: Change me to pub/sub style
         showInfo(`Account funded with MANA`);
         setAccountCreationStatus(APPROVING_MARKETPLACE);
       };
@@ -82,7 +84,8 @@ export class EthereumSignUpScreen extends React.Component {
           status: APPROVING_MARKETPLACE,
           actionId,
         });
-        await action.waitForNonceToUpdate();
+        await action.waitForOneConfirmation();
+        // TODO: Change me to pub/sub style
         showInfo(`Marketplace approved`);
         setAccountCreationStatus(FUNDING_WITH_MANA);
       };

--- a/shared/.gitignore
+++ b/shared/.gitignore
@@ -1,0 +1,1 @@
+node_modules/**/*

